bin/test-storybook.js

```diff
@@ -108,12 +108,30 @@ async function fetchStoriesJson(url) {
   return tmpDir;
 }
 
+function ejectConfiguration () {
+  const origin = path.resolve(__dirname, '../playwright/test-runner-jest.config.js')
+  const destination = path.resolve('test-runner-jest.config.js')
+  const fileAlreadyExists = fs.existsSync(destination)
+  
+  if(fileAlreadyExists) {
+    console.log('[test-runner] Found existing file, overriding...')
```

Comment:
Should we error here instead?

Reply:
I thought this could be further improved with a prompt for yes/no to override. Do we want to make it fail?

Reply:
I think we should just tell them to delete the file & re-run

Reply:
Prompt if you're feeling extra fancy 😄

Reply:
Sounds good. Done!